desktop/Cargo.toml

```diff
@@ -14,10 +14,10 @@ workspace = true
 clap = { workspace = true }
 cpal = "0.15.3"
 egui = { workspace = true }
-egui_extras = { version = "0.27.2", features = ["image"] }
-egui-wgpu = { version = "0.27.2", features = ["winit"] }
+egui_extras = { git = "https://github.com/emilk/egui.git", branch = "master" , features = ["image"] }
```

Comment:
Can we pin this to a revision so that if any api changes on master, anyone who doesn't explicitly run with  `--locked` won't break suddenly?

Reply:
Of course, good idea!